### PR TITLE
Add a class to <body> that's platform dependent so we can have platfo…

### PR DESCRIPTION
**Pull request by jsantell**
…rm-specific styling. r=vporof,phlsa

### CODE REVIEW

File: app/ui/browser/index.jsx [hunk 1]
```diff
@@ -97,3 +98,10 @@ ipcRenderer.on('user-agent-service-info', (_, { port, version, host }) =>
 // Set max listeners to Infinity, because each new tab will listen to more
 // events, and we'll intentionally go beyond the 10 listener default.
 ipcRenderer.setMaxListeners(Infinity);
+
+// Add platform as a class on <body> for platform-specific styling. Possible values:
+// 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32'
+const platform = os.platform();
+const platformClass = platform === 'darwin' ? 'osx' :
+                      platform === 'win32' ? 'win' : 'linux';
```
Comment (victorporof) [inline app/ui/browser/index.jsx]: Not sure it's 100% correct to assume third option always linux. Let's go with another conditional branch here and use "other" or something.

Comment (jsantell) [inline app/ui/browser/index.jsx]: Fair

**Comment by victorporof** [comment]
@jsantell planning to land this soon?

**Comment by victorporof** [comment]
@jsantell ping

**Pull request closed without merging**